src/rpc/misc.cpp

```diff
@@ -348,9 +348,16 @@ static RPCHelpMan verifymessage()
         throw JSONRPCError(RPC_TYPE_ERROR, "Address does not refer to key");
     case MessageVerificationResult::ERR_MALFORMED_SIGNATURE:
         throw JSONRPCError(RPC_TYPE_ERROR, "Malformed base64 encoding");
+    case MessageVerificationResult::ERR_POF:
+        throw JSONRPCError(RPC_TYPE_ERROR, "Proof of funds require UTXO set access to verify"); // TODO: get access to UTXO set / mempool to handle this, then remove this error code?
+    case MessageVerificationResult::INCONCLUSIVE:
+        return false; // TODO: switch to a string based result? mix bool and strings?
```

Comment:
Why not just throw in these other cases?

Reply:
Because these aren't errors? But yeah, that would be one way to return a message. I think I'd prefer to expand the returned value from simple bool to true|false|"inconclusive", but not sure that would be appreciated.

Reply:
Some RPCs return `false` as the result and an error message field. Others just throw with an error code and message. It's an error in the vague sense that the RPC failed to verify what it was asked to verify.

Reply:
OK, but what about "OK_TIMELOCKED"?